i18n-ja-release-8.5: translate doc changes from pingcap/docs release-8.5 on 2026-07-01 (#23189)
pingcap/docs: tidb-cloud/set-up-private-endpoint-connections-serverless.md

AWS VPC 設定で DNS ホスト名と DNS 解決の両方が有効になっていることを確認してください。 [AWS マネジメントコンソール](https://console.aws.amazon.com/)で VPC を作成すると、デフォルトでは無効になります。

## エンドポイントモデルを選択する {#choose-an-endpoint-model}

TiDB Cloudプランに応じて、適切なプライベートエンドポイントモデルを選択してください。

> - {{{ .starter }}} インスタンス、または2026年7月1日より前に作成された {{{ .essential }}} インスタンスの場合は、[**endpoint shared model**](#set-up-a-private-endpoint-with-aws-endpoint-shared-model) を使用します。このモデルでは、同じAWSリージョンおよびVPC内の複数の {{{ .starter }}} または {{{ .essential }}} インスタンスで、1つのプライベートエンドポイントを共有できます。

Reviewer: ```suggestion
- {{{ .starter }}} インスタンス、または2026年7月1日より前に作成された {{{ .essential }}} インスタンスの場合は、[**エンドポイント共有モデル**](#set-up-a-private-endpoint-with-aws-endpoint-shared-model) を使用します。このモデルでは、同じAWSリージョンおよびVPC内の複数の {{{ .starter }}} または {{{ .essential }}} インスタンスで、1つのプライベートエンドポイントを共有できます。
```

> - 2026年7月1日以降に作成された {{{ .essential }}} インスタンスの場合は、[**endpoint exclusive model**](#set-up-a-private-endpoint-with-aws-endpoint-exclusive-model) を使用します。このモデルでは、各 {{{ .essential }}} インスタンスが専用のスタンドアロンプライベートエンドポイントを使用します。このモデルでは接続時に [account prefix](/tidb-cloud/select-cluster-tier.md#user-name-prefix) を含める必要がありませんが、各 {{{ .essential }}} インスタンスごとに設定手順を繰り返す必要があります。

Reviewer: ```suggestion
- 2026年7月1日以降に作成された {{{ .essential }}} インスタンスの場合は、[**エンドポイント占有モデル**](#set-up-a-private-endpoint-with-aws-endpoint-exclusive-model) を使用します。このモデルでは、各 {{{ .essential }}} インスタンスが専用のスタンドアロンプライベートエンドポイントを使用します。このモデルでは接続時に [アカウントプレフィックス](/tidb-cloud/select-cluster-tier.md#user-name-prefix) を含める必要がありませんが、各 {{{ .essential }}} インスタンスごとに設定手順を繰り返す必要があります。
```

    >
    > AWSリージョン内の各VPCにつき、作成する必要があるプライベートエンドポイントは1つだけです。このエンドポイントは、同じAWSリージョン内の同じVPCにあるすべてのTiDB Cloud StarterまたはTiDB Cloud Essentialインスタンスで使用できますが、VPC間で共有することはできません。

## AWSでプライベートエンドポイントを設定する（endpoint exclusive model） {#set-up-a-private-endpoint-with-aws-endpoint-exclusive-model}

> **Note:**
>

> > 現在、endpoint exclusive model は、一部のAWSリージョンにおいて、2026年7月1日以降に作成された {{{ .essential }}} インスタンスでのみ利用できます。お使いのインスタンスで利用できない場合は、代わりに [endpoint shared model](#set-up-a-private-endpoint-with-aws-endpoint-shared-model) を使用できます。

Reviewer: ```suggestion
> 現在、エンドポイント占有モデルは、一部のAWSリージョンにおいて、2026年7月1日以降に作成された {{{ .essential }}} インスタンスでのみ利用できます。お使いのインスタンスで利用できない場合は、代わりに [endpoint shared model](#set-up-a-private-endpoint-with-aws-endpoint-shared-model) を使用できます。
```

> endpoint exclusive model では、各 {{{ .essential }}} インスタンスが専用のスタンドアロンプライベートエンドポイントを使用します。このモデルでは接続時に [account prefix](/tidb-cloud/select-cluster-tier.md#user-name-prefix) を含める必要がありませんが、各 {{{ .essential }}} インスタンスごとに設定手順を繰り返す必要があります。

Reviewer: ```suggestion
エンドポイント占有モデルでは、各 {{{ .essential }}} インスタンスが専用のスタンドアロンプライベートエンドポイントを使用します。このモデルでは接続時に [アカウントプレフィックス](/tidb-cloud/select-cluster-tier.md#user-name-prefix) を含める必要がありませんが、各 {{{ .essential }}} インスタンスごとに設定手順を繰り返す必要があります。
```

> exclusive model を使用して {{{ .essential }}} インスタンスにプライベートエンドポイント経由で接続するには、次の手順を実行します。

Reviewer: ```suggestion
占有モデルを使用して {{{ .essential }}} インスタンスにプライベートエンドポイント経由で接続するには、次の手順を実行します。
```
